building/build-debs/homeworld-admin-tools/src/command.py

```diff
@@ -10,18 +10,21 @@ def fail(message: str) -> None:
 
 
 def mux_map(desc: str, mapping: dict):
-    def usage(err: str = "no command") -> None:
+    def usage(err: str = "no command", loud_err: bool = False) -> None:
```

Comment:
Seems like a rather convoluted way to achieve something simple. Maybe take error handling out of usage (so that the function's only job is to print the command help sheet)? The two of only three times when usage is called with an error message have different error messages anyway.

Reply:
That is... a very good idea and I'm not sure why I didn't do that.